docs/user/en/getting_started/walkthrough2/ExportingtoShapefile.rst

```diff
@@ -5,7 +5,7 @@ In addition to editing content from many sources, you can also export content to
 
 * With **Tasmania roads** selected, choose :menuselection:`File --> Export` from the menu bar
 
-* Select the **Resource to Shapefile** and press :guilabel:`Next`
+* Select the **All Features to Shapefile** and press :guilabel:`Next`
 
   .. image:: images/100000000000020D000002265A8298F4.png
```

Comment:
I guess the image still shows "Resource to Shapefile", just open uDig and take a screenshot to replace image ;)

Reply:
@sschulz92 going to replace the image soon and remove test-case for now